NodeManager.cpp

```diff
@@ -1955,6 +2146,14 @@ int NodeManager::registerSensor(int sensor_type, int pin, int child_id) {
       return registerSensor(new SensorDHT(child_id,pin,dht,SensorDHT::HUMIDITY,dht_type));
     }
   #endif
+  #if MODULE_AM2320 == 1
+    else if (sensor_type == SENSOR_AM2320) {
+      AM2320* th = new AM2320();
```

Comment:
If new AM2320() will be placed in the constructor you can get rid of this that will be unused